src/Drivers/Database.php

```diff
@@ -105,9 +111,10 @@ public function get($token)
 
     /**
      * @param $userId
-     * @return bool|AuthToken
+     * @return AuthToken
+     * @throws Exception
      */
-    public function generate($userId)
+    public function generate($userId): AuthToken
     {
         $authToken = new AuthToken();
         $authToken->setUserId($userId);
```

Comment:
In line 121, please change `parent::generateToken()` to `$this->generateToken()`